### PR TITLE
gif has been added to readme, index rst and local

### LINKED ISSUE #34: Create gifs for setup and run in the READMEs
**Reason**
It makes the project accessible and intuitive for first users

**Design**

Make sure that no personal info is being recorded => fresh browser window, in "no-distraction" mode (e.g. F11 in Firefox)

**GIF 1** 
1. run cookiecutter 
2. bake a new project with some "interesting" naming (we can discuss what it means)
3. cd to repo folder
4. run `make` and wait Jupyter URL is ready. Run with a) the images already downloaded, b) run once before so that we don't wait for db init and upgrades
5. Click the link which opens Jupyter lab in a browser 
5.1 Run mlflow example notebook
6. Open http://localhost:5000/ to connect to `mlflow` and show that the experiment is recorded
7. Save the gif under `images/`
8. Put the gif under "running a baked project" section in the `README`

**GIF 2** 
1. Cut the screencast above and start from step **4**
2. Save the gif under `{{cookiecutter: repo_slug}}/images`
3. Put it in the running the services in `{{cookiecutter: repo_slug}}/README.md`

**Acceptance Criteria**
- Reviewed and liked (:stuck_out_tongue:) by @sertansenturk 

**Outcome**
- The gifs saved to the repo and added to the README

### PR DESCRIPTION
**Pull request by halil-erdogan**
Installation gif has been added to repo.
## Description

- We are adding installation gif into readme, rst and also repo. (closes #34 )
## Motivation and Context

- It will be helpful to newcomer to understand how should they start to use the repo.
## How Has This Been Tested?

- I have checked output visually.
## Types of changes

- [x] Documentation (extending or improving to describe the repo)

## Checklist

- [x] Docs have been added / updated

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -58,3 +58,5 @@ In a terminal, run the following:
    * :ref:`genindex`
    * :ref:`modindex`
    * :ref:`search`
+
+.. image:: ./images/installation.gif
```
Comment (sertansenturk) [inline docs/index.rst]: this should be at the end of quickstart section, not at the end of the file.

File: README.md [hunk 2]
```diff
@@ -48,6 +48,7 @@ To build and start the Docker stack, run:
 cd /{ base_folder }/{{ cookiecutter.repo_slug }}
 make
 ```
+![alt text](./docs/images/installation.gif)
```
Comment (sertansenturk) [inline README.md]: put an empty line in between

**Review by sertansenturk — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (halil-erdogan) [reply README.md]: Done.

File: README.md [hunk 3]
```diff
@@ -49,6 +49,8 @@ cd /{ base_folder }/{{ cookiecutter.repo_slug }}
 make
 ```
 
+![alt text](./docs/images/installation.gif)
```
Comment (sertansenturk) [inline README.md]: Please, change the "alt text" with something meaningful like "quickstart"

File: docs/index.rst (hunk 1 above)
Comment (halil-erdogan) [reply docs/index.rst]: I did it. 💯

File: README.md (hunk 3 above)
Comment (halil-erdogan) [reply README.md]: Done.

File: docs/index.rst (hunk 1 above)
Comment (sertansenturk) [inline docs/index.rst]: Can you specify `alternative text` and size options as well? See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/images.html

Comment (halil-erdogan) [reply docs/index.rst]: Done.

**Review by sertansenturk — APPROVED** [review]

**Pull request merged**